packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -65,7 +65,7 @@ export class Breadcrumb extends BaseComponent<IBreadcrumbProps, any> {
   }
 
   private _onRenderBreadcrumb = (data: IBreadCrumbData) => {
-    const { className, ariaLabel, onRenderItem = this._onRenderItem, overflowAriaLabel } = data.props;
+    const { className, ariaLabel, dividerAs, onRenderItem = this._onRenderItem, overflowAriaLabel } = data.props;
```

Comment:
Hmm. Why not default dividerAs to Icon.

```ts
const { dividerAs = Icon } = props;
```

Reply:
I originally wrote it this way actually but it seemed messier and harder to follow to put it default icon in a constant like so:
```
<Icon
    className={ css('ms-Breadcrumb-chevron', styles.chevron) }
    iconName={ getRTL() ? 'ChevronLeft' : 'ChevronRight' }
/>
```
Or are you suggesting something else?

Reply:
I am suggesting this:

```
const { dividerAs: Divider = Icon } = props;

return ( 
 ...
  <Divider className={ ... } iconName={ ... } />
...
);
```